vars/sendMetricsMsg.groovy

```diff
@@ -26,7 +26,10 @@ def call(Map metricsMap) {
         def params = []
         if (metricsMap['service']['params']) {
             metricsMap['service']['params'].each {
-                params.add("\"$it\"")
+                println(it.getClass())
```

Comment:
I don't think we want a println in production code. Maybe this is left over from debugging?